src/WebJobs.Script.WebHost/Security/IScriptSecretSerializer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    internal interface IScriptSecretSerializer
+    {
+        int SupportedFormatVersion { get; }
```

Comment:
I see you're building out versioning support. However, we don't have any such explicit versioning for any of our other non secret function files (host.json, function.json, etc. Wondering why we need this here.